test/jaxp/javax/xml/jaxp/unittest/common/config/files/catalog2.properties

```diff
@@ -1,13 +1,13 @@
 ################################################################################
 #       XML Library (java.xml) Configuration File
 #
-# This file is in java.util.Properties format and typically located in the conf 
```

Comment:
These spaces at the end of comment lines has crept in since I cleaned all such out in [JDK-8298047](https://bugs.openjdk.org/browse/JDK-8298047). It's a good example of why we need the jcheck verification to keep this from regressing once more.